agent/envoy_bootstrap/envoy_bootstrap.go

```diff
@@ -77,6 +77,8 @@ const (
 	GRPC_MAX_PINGS_WITHOUT_DATA = 0
 	GRPC_KEEPALIVE_TIME_MS      = 10000
 	GRPC_KEEPALIVE_TIMEOUT_MS   = 20000
+	GRPC_INITIAL_BACKOFF_MS     = 10000
+	GRPC_MAX_BACKOFF_MS         = 10000
```

Comment:
I understand that we'll hit max more quickly with a 10 second initial backoff instead of a 1 second, however I think this max backoff is too aggressive. In the case of a widespread issue contacting the ADS server we could see a lot of unhelpful (and potentially harmful) traffic here. 

I'm inclined to leave the max value at it's default. If the multiplier were configurable I'd say we should lower that, but since it's not I don't think we have a really good way to limit backoff without potentially causing wider impact.